.github/workflows/asv-initial.yml

```diff
@@ -0,0 +1,58 @@
+name: asv-benchmarks
+
+on:
+  push:
+    branches: 
+      - asv-ci
```

Comment:
This is still here as your fork has this branch?

Reply:
Yes. I can track if workflow runs for new commits in this pr are successful, or I broke smth